src/magnetVacuumFF.cpp

```diff
@@ -270,6 +271,33 @@ static Ref_t create_detector(Detector& det, xml_h e, SensitiveDetector /* sens *
 
   //----------------------------------------------------
 
+  //--------------------------------------------------------------
+  //make and place vacuum volume after the FF detector array up to end of the world
+  //--------------------------------------------------------------
+  if (make_B2pf_EW_Vacuum) {
+
+    int pieceIdx           = numMagnets - 1; // last B2PF magnet
+    std::string piece_name = Form("GapVacuum%d", numGaps + numMagnets + 1);
+    double endGapLength    = (10000.0 - z_end[pieceIdx]) / cos(rotation_magnet[pieceIdx]);
+    double endGap_z        = 0.5 * endGapLength * cos(rotation_magnet[pieceIdx]) + z_end[pieceIdx];
+    double endGap_x        = 0.5 * endGapLength * sin(rotation_magnet[pieceIdx]) + x_end[pieceIdx];
+
+    Tube vacuum_endWorld(piece_name, 0.0, 4 * radii_magnet[pieceIdx],
+                         endGapLength / 2); // make larger tube than inner magnet radius
+    Volume vpiece(piece_name, vacuum_endWorld, m_Vac);
+    sdet.setAttributes(det, vpiece, x_det.regionStr(), x_det.limitsStr(),
+                       "InvisibleNoDaughters"); // make invisible instead of AnlBlue
+
+    auto pv = assembly.placeVolume(vpiece, Transform3D(RotationY(rotation_magnet[pieceIdx]),
+                                                       Position(endGap_x, 0.0, endGap_z)));
+
+    pv.addPhysVolID("sector", 1);
+
+    DetElement de(sdet, Form("sector%d_de", numGaps + numMagnets + 1), 1);
+    de.setPlacement(pv);
```

Comment:
```suggestion
```

Not needed for dummy volume.

Reply:
I copy paste this block from a different place. Does it mean that the dummy volume needs to be removed everywhere?  
https://github.com/eic/epic/blob/d6746500443b3e21bee7991998e2e12095d15006/src/magnetVacuumFF.cpp#L210-L213
https://github.com/eic/epic/blob/d6746500443b3e21bee7991998e2e12095d15006/src/magnetVacuumFF.cpp#L237-L240
https://github.com/eic/epic/blob/d6746500443b3e21bee7991998e2e12095d15006/src/magnetVacuumFF.cpp#L266-L269

Reply:
Yes. None of those make sense, could you please remove.

Reply:
ok, removed